Utils/src/PhotonIDisoProducer.cc

```diff
@@ -182,17 +182,18 @@ PhotonIDisoProducer::produce(edm::StreamID, edm::Event& iEvent, const edm::Event
   // Initializing effective area to be used 
   // for rho corrections to the photon isolation
   // variables. 
+  // Spring16 EA are used. Spring15 EA are commented out.
```

Comment:
do not comment out old lines of code, just delete them. git has the old lines in the history already.

Reply:
still need to remove this comment